docs/maintain-errors.md

```diff
@@ -38,11 +38,12 @@ table below.
+| Future             | Transaction nonce too high, i.e. it's "from the future".                                                     | Reduce the nonce to +1 of current nonce. Check current nonce by inspecting the address you're using to send the transaction.                                                                                                                                                                                                                                                              |
+| Stale              | Transaction nonce too low.                                                                                   | Increase the nonce to +1 of current nonce. Check current nonce by inspecting the address you're using to send the transaction.                                                                                                                                                                                                                                                            |
+| ExhaustsResources  | There aren't enough resources left in the current block to submit this transaction.                          | Try again in the next block.                                                                                                                                                                                                                                                                                                                                                              |
+| Payment            | Unable to pay for TX fee.                                                                                    | You might not have enough free balance to cover the fee this transaction would incur.                                                                                                                                                                                                                                                                                                     |
+| Temporarily banned | The transaction is temporarily banned.                                                                       | The tx is already in pool. Try on a different node and wait.                                                                                                                                                                                                                                                                                                                              |
```

Comment:
```suggestion
| Temporarily banned | The transaction is temporarily banned.                                                                       | The tx is already in pool. Either try on a different node, or wait to see if the initial transaction goes through.                                                                                                                                                                                                                                                                                                                              |
```

Reply:
What change is new in this suggestion? I don't see a difference 🤔

Reply:
Scroll to the right =)

Reply:
AH i didn't see the scrollbar 😂